caddy_chatbot/src/caddy_core/components.py

```diff
@@ -371,15 +368,43 @@ def check_existing_call(caddy_message) -> Tuple[Dict[str, Any], bool]:
     return module_values, survey_complete
 
 
+def reword_advisor_message(message: str) -> str:
```

Comment:
Looks interesting -@kurits to advise, should this be moved to where the other promtps are stored?

Reply:
It's great! Have just tested locally/google chat - I think if we may want to modify the prompt in the future to extract different info it would be good to add to the caddy prompts table and read back out with the get_prompt function - it can be imported from caddy_core.utils.prompt